rotkehlchen/inquirer.py

```diff
@@ -1267,6 +1271,23 @@ def find_hop_lp_price(self, lp_token: EvmToken) -> Price | None:
             log.error(f'Failed to query virtual price of Hop lp token {lp_token.evm_address!s} in {evm_manager.node_inquirer.chain_name!s}. {e!s}')  # noqa: E501
             return ZERO_PRICE
 
+    def find_umami_vault_price(self, token: EvmToken) -> Price | None:
```

Comment:
what's the point ofthis function? Can't you just call `get_umami_vault_token_price(` directly in the one place you use it? Please don't do useless things and spend effort for nothing. We are wasting time.